### ISSUE TITLE
Doesn't detect modules used with require.resolve('module')

### ISSUE THREAD

**Issue by maxkueng** [dylang/npm-check#167]
I'm using the `emojify.js` module as follows:

```js
const emojiDir = path.resolve(path.dirname(require.resolve('emojify.js')), '..', 'images', 'basic')
```

But `npm-check` keeps asking me whether I'm really using this module.

**Comment by septs** [comment]
@dylang

**Comment by SEAPUNK** [comment]
Bump

**Comment by LinusU** [comment]
PR welcome, this should be fixed 👍
